New Music Trivia List

japandotorg opened the pull request:
### Description of the changes

Adds a trivia list for the Music category, ideas/questions has been taken from multiple sources online.


### Have the changes in this PR been tested?

Yes

japandotorg committed:
new trivia category - music

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+- ABBA (with SOS)
+Which artist released an album and movie under the same title of Get Rich Or Die Tryin'?:
+- 50 Cent
+Which US rapper died on September 13th 1996 after being shot in a drive-by shooting after watchin Mike Tysons comeback fight 7 days earlier?:
```
```suggestion
Which US rapper died on September 13th 1996 after being shot in a drive-by shooting after watching Mike Tysons comeback fight 7 days earlier?:
```

japandotorg committed:
fix

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+- ABBA (with SOS)
+Which artist released an album and movie under the same title of Get Rich Or Die Tryin'?:
+- 50 Cent
+Which US rapper died on September 13th 1996 after being shot in a drive-by shooting after watching Mike Tysons comeback fight 7 days earlier?:
```
```suggestion
Which US rapper died on September 13th 1996 after being shot in a drive-by shooting after watching Mike Tyson's comeback fight 7 days earlier?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+- Rihanna
+What nationality was Mozart?:
+- Austrian
+Which boy`s name was the title of the debut album released by Amy Winehouse?:
```
```suggestion
Which boy's name was the title of the debut album released by Amy Winehouse?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+- Kiki Dee
+What is the common link between The Dave Clark 5, Fat Larry's Band and Harold Melvin and the Blue Notes?:
+- All three are named after their drummer
+George Bernard Shaw`s play `Pygmalion` was adapted to become which musical?:
```
```suggestion
George Bernard Shaw`s play 'Pygmalion' was adapted to become which musical?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+Which band had hits with 'Linger' and 'Dreams'?:
+- The Cranberries
+How is the double act of David Peacock and Charles Hodges better known?:
+- Chas `n` Dave
```
```suggestion
- Chas 'n' Dave
- Chas & Dave
- Chas and Dave
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+- The Move
+Who sang the first line of the USA for Africa's "We Are The World"?:
+- Lionel Richie
+Which American soul singer`s dad played football for Glasgow Celtic ?:
```
```suggestion
Which American soul singer's dad played football for Glasgow Celtic ?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+- Gil Scott Heron
+Which Paul released his first single "I Confess" when he was 14 and had written over 200 songs by the time he was 21?:
+- Paul Anka
+What was the title of Elvis Presley`s first British number 1?:
```
```suggestion
What was the title of Elvis Presley's first British number 1?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+- Beastie Boys
+Justin Timberlake was formerly a vocalist for which band?:
+- N Sync
+Who was lead singer for 70s glam rock band `The Sweet`?:
```
```suggestion
Who was the lead singer for the 70's glam rock band 'The Sweet'?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+- Radiohead
+Which Spice Girl realeased the album "Schizophonic"?:
+- Geri Halliwell
+In the song `The Twelve Days Of Christmas', how what did my true love give to me on the 12th day?:
```
```suggestion
In the song 'The Twelve Days Of Christmas', what did my true love give to me on the 12th day?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+What is the name of the 90s song which includes the line "I swear I left her by the river"?:
+- Hazard (by Richard Marx)
+- Hazard by Richard Marx
+Which band had a UK number 2 hit in 1967 with `Hole In My Shoe`, which was covered in 1985 by Neil from the TV show "The Young Ones"?:
```
```suggestion
Which band had a UK number 2 hit in 1967 with 'Hole In My Shoe', which was covered in 1985 by Neil from the TV show "The Young Ones"?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+- The Barber Of Seville
+Who had a top ten hit in 1987 called "Shattered Dreams"?:
+- Johnny Hates Jazz
+Which group had a 2001 hit with a cover version of Billy JoelÃƒÂ¢Ã¢â€šÂ¬Ã¢â€žÂ¢s `Uptown Girl`?:
```
```suggestion
Which group had a 2001 hit with a cover version of Billy Joel's 'Uptown Girl'?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+- Saving All My Love For You
+What was Bob Marley's backing band called?:
+- The Whalers
+Which band`s first top ten single was "Love Is Life" in 1970?:
```
```suggestion
Which band's first top ten single was \"Love Is Life\" in 1970?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+- The Whalers
+Which band`s first top ten single was "Love Is Life" in 1970?:
+- Hot Chocolate
+Who`s 1978 debut single was called "Sultans Of Swing"?:
```
```suggestion
Who had a 1978 debut single called "Sultans Of Swing"?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+- 1950s (1956)
+- 1950s
+- 1956
+Which of Michael Jackson`s albums sold most copies in the UK? "Thriller" or "Bad"?:
```
```suggestion
Which of Michael Jackson's albums sold most copies in the UK? \"Thriller\" or \"Bad\"?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+- 10 cc
+Who recorded the 1994 album "Return Of The Space Cowboy"?:
+- Jamiroquai
+Which Lightning Seeds song includes the line "Oh you fool, you`ve got me started"?:
```
```suggestion
Which Lightning Seeds song includes the line "Oh you fool, you've got me started"?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+- The trumpet
+Who had a Christmas number one in 1974 with "Lonely This Christmas"?:
+- Mud
+In 2002, who was the only musician to be voted in the top ten in BBC`s "Great Briton's" poll?:
```
```suggestion
In 2002, who was the only musician to be voted in the top ten in BBC's \"Greatest Britons\" poll?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+- The Human League
+Which band had a hit with "He Ain't Heavy, He's My Brother" in both 1969 and 1988?:
+- The Hollies
+Which band had a hit with "He Ain't Heavy, He's My Brother" in both 1969 and 1988?:
```
Remove lines 88 and 89 as they are duplicates of 86 and 87.

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+Which Zager and Evans hit single of 1969 had the words Exordium and Terminus in brackets in the title?:
+- In the year 2525
+- 2525
+Which famous comedian made a record called 'please mister custer' in 1960?:
```
It seems like this item is both a song title and a record title, but it's styled as "Mr. Custer" and not 'please mister custer'. I am leaving this as something for you to manually review instead of a suggestion as I may not have been looking at the right song/record that you were thinking of.

japandotorg replied:
Okay so after researching a bit I think changing it to "Mr. Custer" would be appropriate.

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+- The England World Cup Squad
+Which instrument was Dizzy Gillespie famous for playing?:
+- The trumpet
+Who sang lead vocals on Australian group Python Lee Jackson's 1972 hit "In a broken dream"?:
```
```suggestion
Who sang lead vocals on Australian group Python Lee Jackson's 1972 hit "In a Broken Dream"?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+What was the title of Elvis Presley`s first British number 1?:
+- All Shook Up
+Which artist has had number one singles in the UK as a solo artist, as part of a duo, as part of a trio, as part of a quartet and as part of a group with more than four members?:
+- Sir Paul McCartney
```
```suggestion
- Sir Paul McCartney
- Paul McCartney
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+- Bono
+In which year did Elvis Presley die?:
+- 1977
+The "No Sleep Till Brooklyn" was a hit single for which group in 1987?:
```
```suggestion
The song "No Sleep Till Brooklyn" was a hit single for which group in 1987?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+- Whitney Houston & Shania Twain
+- Whitney Houston, Shania Twain
+- Whitney Houston Shania Twain
+Which world famous star is a member of the band "30 odd foot of grunts"?:
```
```suggestion
Which world famous star is a member of the band "30 Odd Foot of Grunts"?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+- Whitney Houston Shania Twain
+Which world famous star is a member of the band "30 odd foot of grunts"?:
+- Russel Crowe
+What is the name of Iron Maidens mascot who appears on their album covers and who has also featured in a video game?:
```
```suggestion
What is the name of Iron Maidens' mascot who appears on their album covers and who has also featured in a video game?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+- 2
+Who was the lead singer in The Police?:
+- Sting
+Now better known as an actress, who was Britains entry in the Eurovision Song Contest in 1991 with the song "A Message To Your Heart"?:
```
```suggestion
Now better known as an actress, who was Britain's entry in the Eurovision Song Contest in 1991 with the song "A Message To Your Heart"?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+- Sting
+Now better known as an actress, who was Britains entry in the Eurovision Song Contest in 1991 with the song "A Message To Your Heart"?:
+- Samantha Janus
+Who was Britains entry in the Eurovision Song Contest in 1968 with the song "Congratulations"?:
```
```suggestion
Who was Britain's entry in the Eurovision Song Contest in 1968 with the song "Congratulations"?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+- Cliff Richard
+Who sang with KLF on their 1991 hit single "Justified And Ancient"?:
+- Tammy Wynette
+Who was Britains entry in the Eurovision Song Contest in 1964 with the song "I Love The Little Things"?:
```
```suggestion
Who was Britain's entry in the Eurovision Song Contest in 1964 with the song "I Love The Little Things"?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+- Herman's Hermits
+Whose first solo number one was called "Forever Love" in 1996?:
+- Gary Barlow
+Who sang the christmas song "Driving Home For Christmas"?:
```
```suggestion
Who sang the Christmas song "Driving Home For Christmas"?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+- Sheena Easton
+Les McKeown was the lead singer with which Scottish pop group?:
+- The Bay City Rollers
+The Dutch duo of Ray Slijngaard and Anita Dels, who had 8 top ten hits in the 90s, were better known by what name?:
```
```suggestion
The Dutch duo of Ray Slijngaard and Anita Dels, who had 8 top ten hits in the 90's, were better known by what name?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+Roisin Murphy is the singer in which band?:
+- Moloko
+What instrument was Louis Armstrong most famous for playing?:
+- The trumpet.
```
Remove line 309 as it's the same as 310

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+- Clarinet
+Who played the villain Howard Payne in the film Speed?:
+- Dennis Hopper
+What was the name of country music's first superstar who died at the age of 29 from drink and drug abuse?:
```
```suggestion
What was the name of country music's first superstar who died at the age of 29 from drinking and drug abuse?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+- Pink
+In 2003, who won the Best Male Solo Artist award at the Brits?:
+- Robbie Williams
+What is the name of the 90s song which includes the line "I swear I left her by the river"?:
```
```suggestion
What is the name of the 90's song which includes the line "I swear I left her by the river"?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+- Anarchy in the UK
+Which female vocalist released the album "Drag"?:
+- K. D. Lang
+What brand of beer does Homer Simpson drink?:
```
How is line 430 and 431 related to music?

japandotorg replied:
It's related to a song named "[The Homer Simpson Beer Song](https://www.youtube.com/watch?v=TyiTY_jubF8)", but then it's upto you if you'd want me to keep it there or just wipe it off

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,544 @@
+- Don't Stop Moving by S Club 7
+- Don't Stop Moving
+Which song includes the line "She'll make you take your clothes off and go dancing in the rain"?:
+- Livin La Vie Da Loca (by Ricky Martin)
+- Livin' La Vie Da Loca (by Ricky Martin)
+- Livin La Vie Da Loca by Ricky Martin
+- Livin' La Vie Da Loca by Ricky Martin
+- Livin' La Vie Da Loca
+- Livin La Vie Da Loca
```
```suggestion
- Livin' La Vida Loca (by Ricky Martin)
- Livin La Vida Loca (by Ricky Martin)
- Livin' La Vida Loca by Ricky Martin
- Livin La Vida Loca by Ricky Martin
- Livin' La Vida Loca
- Livin La Vida Loca
```

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
ty aika

japandotorg committed:
fixed

"Mr. Custer" would be more appropriate than "please mister custer" in respect of the original title of the song.

japandotorg committed:
Out of context for a trivia list

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,542 @@
+By what name is singer Pauline Matthews better known?:
+- Kiki Dee
+What is the common link between The Dave Clark 5, Fat Larry's Band and Harold Melvin and the Blue Notes?:
+- All three are named after their drummer
```
I feel like this question may be a bit too ambiguous for a trivia game because there's so many different ways people can phrase this answer, it's not like there's a succinct way to say this concept in one or two words... and as you have it now, people will have to say this answer exactly to be correct. What do you think?

japandotorg replied:
yeah you're right, this particular answer is not appropriate for a trivia game and can be said in alot of ways

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,542 @@
+Which group had a hit record in the 1970s, the 1980s and the 1990s and each time they had a completely different line up?:
+- The England World Cup Squad
+Which instrument was Dizzy Gillespie famous for playing?:
+- The trumpet
```
```suggestion
- The trumpet
- trumpet
```

aikaterna replied:
```suggestion
- The trumpet
- trumpet
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,542 @@
+- The Drifters
+Who had a hit in 1984 with "99 Red Balloons"?:
+- Nena
+Who had a 1983 hit with "Too Shy"?:
```
Lines 175 and 176 are a repeat of 100 and 101

aikaterna replied:
Lines 175 and 176 are a repeat of 100 and 101

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,542 @@
+- 24
+Which popstar was actor Emilio Estevez formerly married to?:
+- Paula Abdul
+For what is Michael Eavis best known in the music world?:
```
```suggestion
What is Michael Eavis best known for in the music world?:
```

aikaterna replied:
```suggestion
What is Michael Eavis best known for in the music world?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,542 @@
+Which record label did Sam Phillips sell Elvis Presley to in 1955?:
+- R.C.A.
+- RCA
+Which UK group were banned from entering the U.S. for unspecified reasons from 1965 to 1969?:
```
```suggestion
Which UK group was banned from entering the U.S. for unspecified reasons from 1965 to 1969?:
```

aikaterna replied:
```suggestion
Which UK group was banned from entering the U.S. for unspecified reasons from 1965 to 1969?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,542 @@
+- Oasis's Wonderwall
+In which month of the year do crowds visit Graceland to honour Elvis on the anniversary of his death?:
+- August
+How many black keys are there on the a standard piano?:
```
```suggestion
How many black keys are there on a standard piano?:
```

aikaterna replied:
```suggestion
How many black keys are there on a standard piano?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,542 @@
+- August
+How many black keys are there on the a standard piano?:
+- 36
+As at May 2001, what is Pink Floyd's only UK number one single?:
```
```suggestion
As of May 2001, what is Pink Floyd's only UK number one single?:
```

aikaterna replied:
```suggestion
As of May 2001, what is Pink Floyd's only UK number one single?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,542 @@
+- 2 Unlimited
+Which solo artist had the second and third best-selling albums of the 1980s in the UK?:
+- Michael Jackson
+Which groups first UK top ten hit was "Follow You, Follow Me" in 1978?:
```
```suggestion
Which group's first UK top ten hit was "Follow You, Follow Me" in 1978?:
```

aikaterna replied:
```suggestion
Which group's first UK top ten hit was "Follow You, Follow Me" in 1978?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,542 @@
+Roisin Murphy is the singer in which band?:
+- Moloko
+What instrument was Louis Armstrong most famous for playing?:
+- The trumpet
```
```suggestion
- The trumpet
- trumpet
```

aikaterna replied:
```suggestion
- The trumpet
- trumpet
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,542 @@
+- Do They Know Its Christmas by Band Aid
+Which instrument was Benny Goodman famous for playing?:
+- Clarinet
+Who played the villain Howard Payne in the film Speed?:
```
This doesn't seem like a music related question.

aikaterna replied:
This doesn't seem like a music related question.

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,542 @@
+- Hank Williams
+What was Billy Ocean's first number one single?:
+- Caribbean Queen
+Which band were accused of poll rigging in 1999, when they won the Brit award for best newcomer?:
```
```suggestion
Which band was accused of poll rigging in 1999 when they won the Brit award for best newcomer?:
```

aikaterna replied:
```suggestion
Which band was accused of poll rigging in 1999 when they won the Brit award for best newcomer?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,542 @@
+- Robbie Williams
+What is the name of the 90's song which includes the line "I swear I left her by the river"?:
+- Hazard (by Richard Marx)
+- Hazard by Richard Marx
```
```suggestion
- Hazard by Richard Marx
- Hazard
```

aikaterna replied:
```suggestion
- Hazard by Richard Marx
- Hazard
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,542 @@
+- Traffic
+Olly Peacock is the drummer with which group?:
+- Gomez
+In which American state is the museum called the Rock and Roll Hall of Fame?:
```
```suggestion
The Rock and Roll Hall of Fame museum is in which American state?:
```

aikaterna replied:
```suggestion
The Rock and Roll Hall of Fame museum is in which American state?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,542 @@
+- Primal Scream
+Which R.E.M. song includes the line "Let's play twister, let's play risk"?:
+- Man On The Moon
+The Pet Shop Boys had a UK Christmas Number 1 in 1987 with a song that was co-written by country star, Willie Nelson. What was the name of the song?:
```
```suggestion
The Pet Shop Boys had a UK Christmas number 1 in 1987 with a song that was co-written by country star Willie Nelson. What was the name of the song?:
```

aikaterna replied:
```suggestion
The Pet Shop Boys had a UK Christmas number 1 in 1987 with a song that was co-written by country star Willie Nelson. What was the name of the song?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,542 @@
+Who had a 1978 debut single called "Sultans Of Swing"?:
+- Dire Straits
+Who played bass guitar in the Beatles?:
+- Paul McCartney
```
```suggestion
- Sir Paul McCartney
- Paul McCartney
```

aikaterna replied:
```suggestion
- Sir Paul McCartney
- Paul McCartney
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,542 @@
+- Doctor Jones
+Which football team do Liam and Noel Gallagher of Oasis support?:
+- Manchester City
+Which pop group comprised Vince Clarke and Alison Moyet?:
```
```suggestion
Which pop group was comprised of Vince Clarke and Alison Moyet?:
```

aikaterna replied:
```suggestion
Which pop group was comprised of Vince Clarke and Alison Moyet?:
```

aikaterna commented on redbot/cogs/trivia/data/lists/music.yaml:
```diff
@@ -0,0 +1,542 @@
+- Sting
+What famous heavy metal band is named after a medieval torture device?:
+- Iron Maiden
+What was the name of No Doubt's hit single which included the line "Don't tell me cos it hurts"?:
```
```suggestion
What was the name of No Doubt's hit single which included the line "Don't tell me 'cause it hurts"?:
```

aikaterna replied:
```suggestion
What was the name of No Doubt's hit single which included the line "Don't tell me 'cause it hurts"?:
```

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Update redbot/cogs/trivia/data/lists/music.yaml

Co-authored-by: aikaterna <20862007+aikaterna@users.noreply.github.com>

japandotorg committed:
Made changes as per Aika's change requests

japandotorg committed:
duplicate removed

aikaterna committed:
add alternative answers/remove duplicates

aikaterna commented:
Thank you for your patience on this.
